packages/react-components/src/Status/index.tsx

```diff
@@ -144,8 +145,24 @@ function Status ({ className, stqueue = [], txqueue = [] }: Props): React.ReactE
     return null;
   }
 
+  const _onDismiss = (): void => {
+    allst.map(s => s.removeItem());
+    alltx.map(t => t.removeItem());
+  };
+
   return (
     <div className={`ui--Status ${className}`}>
+      {allst.length + alltx.length > 1 && (
+        <div className='dismiss'>
```

Comment:
We probably need to split the notifications, i.e. with opacity from the button so the button itself has no opacity. (It is slightly harder to read)

Reply:
I'm not sure I understand this comment. Can you maybe share a screenshot or something so that I understand it better? Thanks!

Reply:
Will re-check the latest version - it may just be a non-issue with extending over the full width. (The initial version was quite hard to read since it is opaque.) However, let's see what the full version + icon looks like, it may just solve itself.

Reply:
Yes, it is fine with the current full width.